registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/verticle/WorkflowInternalActionVerticle.java

```diff
@@ -481,6 +484,19 @@ private void processPauseAndRequestAdditionalInfo(WorkflowInternalActionDTO work
 			workflowActionService.processWorkflowAction(internalRegistrationStatusDtos,
 					WorkflowActionCode.RESUME_PROCESSING.toString());
 		} else {
+			List<AdditionalInfoRequestDto> additionalInfoRequestDtos = additionalInfoRequestService.
+					getAdditionalInfoRequestByRegIdAndProcess(
+						workflowInternalActionDTO.getRid(), workflowInternalActionDTO.getAdditionalInfoProcess());
+			String iteration=env.getProperty("mosip.regproc.workflow-manager.internal.action.max.allowed.iteration." + workflowInternalActionDTO.getAdditionalInfoProcess());

```

Comment:
make this also action-max-allowed.<process_name>

Reply:
fixed